### ISSUE TITLE
Broken Antline Dots in Old Aperutre

### ISSUE THREAD

**Issue by Neightro** [BEEmod/BEE2-items#2563]
I have noticed a lot of antline dots in the Old Aperture style that don't light up. I imagine that this may be intended—that they are just meant to be burnt out. I would, however, rather see this fixed or at least made into a stylevar.

**Comment by villager103** [comment]
Could we also get a similar thing for Overgrown chambers? None of the overgrown chambers in the campaign had broken dots, so can this stylevar also apply to the Overgrown theme?

**Comment by TeamSpen210** [comment]
Both do have this, it's very much intentional. The Old Aperture ones have cracked glass. I have switched it in the development versions though so it doesn't have lots of repeated ones.

**Comment by CreepiX987** [comment]
StyleVar?

**Comment by Neightro** [comment]
@TeamSpen210 Not having so many in a row in a row sounds good. The large groups make the antlines hard to follow where it's dark.

**Comment by Neightro** [comment]
@CreepiX987 Stylevars are customizations for styles. For example, using the office observation room in Old Aperture is a stylevar.
